diff --git a/src/include/pg_config.h.win32 b/src/include/pg_config.h.win32
index dfd6972383..72d1259ffb 100644
--- a/src/include/pg_config.h.win32
+++ b/src/include/pg_config.h.win32
@@ -74,12 +74,15 @@
 
 /* Define to 1 if you have the `ASN1_STRING_get0_data' function. */
 /* #undef HAVE_ASN1_STRING_GET0_DATA */
+#define HAVE_ASN1_STRING_GET0_DATA 1
 
 /* Define to 1 if you have the `BIO_get_data' function. */
 /* #undef HAVE_BIO_GET_DATA */
+#define HAVE_BIO_GET_DATA 1
 
 /* Define to 1 if you have the `BIO_meth_new' function. */
 /* #undef HAVE_BIO_METH_NEW */
+#define HAVE_BIO_METH_NEW 1
 
 /* Define to 1 if you have the `cbrt' function. */
 //#define HAVE_CBRT 1
